### ISSUE TITLE
Compare our performance with the Deep Speech 2 model

### ISSUE THREAD

**Issue by reuben** [mozilla/DeepSpeech#330]

**Comment by reuben** [comment]
WIP in https://github.com/mozilla/DeepSpeech/tree/issue330_ds2

**Comment by reuben** [comment]
Note to self:
1. Remove the manual context and stride implemented in audiofile_to_input_vector.
2. Do some runs with negative wav file size and see if this impacts batch sizes (it should).
3. Batch norm gives us a little bit of room in the learning rate, but maybe we don't have time to explore that thoroughly just yet.

**Comment by reuben** [comment]
Running experiments at home with LapsBM, I:

1. Removed the explicit context
2. Increased batch size from 8 to 120
3. Increased learning rate from 1x10<sup>-4</sup> to 1x10<sup>-3</sup> and still was able to overfit it

**Comment by kdavis-mozilla** [comment]
Closing for lack of activity.
